patches/chrome-browser-importer-importer_list.cc.patch

```diff
@@ -1,22 +1,8 @@
 diff --git a/chrome/browser/importer/importer_list.cc b/chrome/browser/importer/importer_list.cc
-index c8b1156c1be3d4b31c266b723e3a3ea9e50c73e8..ca51773d102873b2d84b77898841f939e71cce52 100644
+index c12f8955c7a608d90b8492b5425958da3e7e429d..f8eb1cfbaec76ac4da3dee4f4cebf9aeaffb7027 100644
 --- a/chrome/browser/importer/importer_list.cc
 +++ b/chrome/browser/importer/importer_list.cc
-@@ -7,9 +7,13 @@
- #include <stdint.h>
- 
- #include "base/bind.h"
-+#include "base/strings/utf_string_conversions.h"
- #include "base/task/post_task.h"
- #include "base/task/task_traits.h"
- #include "base/threading/thread_restrictions.h"
-+#include "base/values.h"
-+#include "brave/common/importer/brave_importer_utils.h"
```

Comment:
is this right? Did you move to chromium_src or are these no longer needed for COOKIES?